src/toil/test/provisioners/aws/awsProvisionerTest.py

```diff
@@ -260,18 +398,50 @@ def launchCluster(self):
         # test that workers have expected storage size
         # just use the first worker
         worker = workers[0]
-        worker = next(wait_instances_running(self.cluster._ctx.ec2, [worker]))
+        worker = next(wait_instances_running(self.cluster._boto2.ec2, [worker]))
         rootBlockDevice = worker.block_device_mapping["/dev/xvda"]
         self.assertTrue(isinstance(rootBlockDevice, BlockDeviceType))
-        rootVolume = self.cluster._ctx.ec2.get_all_volumes(volume_ids=[rootBlockDevice.volume_id])[0]
+        rootVolume = self.cluster._boto2.ec2.get_all_volumes(volume_ids=[rootBlockDevice.volume_id])[0]
         self.assertGreaterEqual(rootVolume.size, self.requestedNodeStorage)
 
     def _runScript(self, toilOptions):
-        runCommand = ['/home/venv/bin/python', '/home/sort.py', '--fileToSort=/home/sortFile']
+        # Autoscale even though we have static nodes
+        toilOptions.extend(['--provisioner=aws', '--batchSystem=mesos',
+                            '--nodeTypes=' + ",".join(self.instanceTypes),
+                            '--maxNodes=' + ",".join(self.numWorkers)])
```

Comment:
Should this explicitly state the `--zone`?

Reply:
I don't think so. None of the other ones do; they should pick up the zone from the instance metadata and/or TOIL_AWS_ZONE.

It does mean you have to set TOIL_AWS_ZONE when running tests manually. But you'd always have to manually pick a zone to run in; I don't think we want to hardcode a testing zone into the tests.